buildenv/jenkins/testJobTemplate

```diff
@@ -348,7 +350,12 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							}
 							choiceParam('PARALLEL', PARALLEL_LIST, "Optional. Parallel mode")
 							stringParam('NUM_MACHINES', NUM_MACHINES, "Optional. Number of machines to run in parallel. Need to be used with PARALLEL=Dynamic")
-							booleanParam('PERSONAL_BUILD', false, "Is this a personal build?")
+							stringParam('TEST_TIME', TEST_TIME, '''Optional. Expected overall test build execution time. Need to be used with PARALLEL=Dynamic.<br>
```

Comment:
Minor: Need**s** to be used versus Need to be used on line 353 and 357

Reply:
updated. Thanks